pyccel/ast/builtin_methods/set_methods.py

```diff
@@ -132,3 +132,30 @@ def __init__(self, set_variable):
         self._precision = set_variable._precision
         self._class_type = set_variable._class_type
         super().__init__(set_variable)
+
+
+class SetPop(SetMethod):
+    """
+    Represents a call to the .pop() method.
+
+    The pop() method pops an element from the set. 
+    It does not take any argument but returns the popped 
+    element. It raises an error if the element is not 
+    present in the set.
```

Comment:
This sentence looks like it has been copy/pasted from the `SetRemove` PR. I don't think it is relevant here.

Reply:
Yes, my bad. I will fix in the next commit. It should be `It raises an error if the set is empty.`

Reply:
Fixed : 35bef02